### DISCUSSION TITLE
2007/06/21/%E5%9F%BA%E5%9B%A0%E5%AF%B9%E4%BA%BA%E7%9A%84%E5%BD%B1%E5%93%8D

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#167] [Announcements]
# 2007/06/21/%E5%9F%BA%E5%9B%A0%E5%AF%B9%E4%BA%BA%E7%9A%84%E5%BD%B1%E5%93%8D

最近在读《心理学》这是一本非常厚的书，估计要读上一阵子。这几天读的章节是进化心理学和行为心理学。里面列举了一些调查结果，和我曾经想像的还是有很大差别的。

https://blog.qizhen.xyz/2007/06/21/%E5%9F%BA%E5%9B%A0%E5%AF%B9%E4%BA%BA%E7%9A%84%E5%BD%B1%E5%93%8D

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Tian
2007年06月27日 19:42 编辑
看来我是基因决定论者，我认为三岁看到老，而三岁是由基因决定的。
不过，环境对人的眼界的影响很大，从而对他的人生决策影响很大。像我这样从农村来的人，人生的最开始眼界是很窄的，看到的路很少，从而选择就少，所做的选择往往就不是最优化的。
另外一方面，概率论是很神奇的。据说两个智商远超普通人的人结为夫妻，他们的小孩往往也还是智商超过普通人，但是超过的程度却不如他的父母。这就是一种极而后返效应，所以不用担心人类的智商若干代以后极端分化:-)

回复
K
2007年06月27日 06:33 编辑
好久没来了，今天偶然又溜达过来了。

回复
竹贤
2007年06月26日 17:54 编辑
生物界经过了亿万年的进化，各种生物的基本基因应该是与自然界相容的，那些不相容的早已经绝灭了。就人类而言，人类的基本基因也应该是如此。当然，就个体而言，个体的许多特征（外貌，体质，性格，以及所谓的种种智商、情商等等…）肯定与基因有关。但我总觉得，基因绝对不是这么一个简单的东西。一方面，除了极少数的因为基因缺陷而引起的基因疾病外，正常基因所表露出来的特征，应该是无所谓好坏之分的。所谓的好坏的区分，大部分是由于历史上人类的无知或偏见。上帝在在给你关上一扇窗的时候，也一定为你打开了另一道门。我们不必羡慕别人打开了的窗户，而一定要寻找到，并充分用好为自己打开了的那道门。另一方面，今天，我们看到的、了解的恐怕也还只是基因的某些显性特征，而更多的隐性特征也许尚未被人类自己所认识。我们不能只被这些显性的特征所左右，而应该积极认识开发那些隐性的特征。我并不否认天才，他们也许就是上帝恩赐给人类的礼物。但更多的天才，也许因为没有机遇，没有被发现，而被埋没消失了。

回复
楠
2007年06月22日 22:46 编辑
我觉得是有道理的，虽然感情上不容易接受，但是基因确实是主导智商的决定性因素。

回复
Wu
2007年06月22日 10:03 编辑
人除了智商，还有很多其他差异的嘛，比如性格、体格等。
但我感情上也难以接受这个结果。比较容易否定后天对人的影响。严重点说，可能会引发种族歧视。
尽信书则不如无书，不可不信，也不可全信。

回复
arrowpig
2007年06月21日 22:15 编辑
这个极端了，个人觉得大家智商差不多。然后就是环境，因为环境能影响人的意识和观念。意识观念差不多，看谁更努力。努力程度差不多，看谁更能坚持…

回复
Wu
2007年06月21日 22:14 编辑
长见识了！
